extensions/blocks/podcast-player/view.js

```diff
@@ -1,19 +1,62 @@
 /* global jetpackPodcastPlayers */
+/**
+ * External dependencies
+ */
+import debugFactory from 'debug';
+import { render, createElement } from '@wordpress/element';
+
 /**
  * Internal dependencies
  */
 import './style.scss';
-import './closest-shim';
-import PodcastPlayer from './podcast-player';
+import PodcastPlayer from './components/podcast-player';
 
+const debug = debugFactory( 'jetpack:podcast-player' );
 const playerInstances = {};
 
+/**
+ * Initialize player instance.
+ * @param {string} id The id of the block element in document.
+ */
 const initializeBlock = function( id ) {
-	// Create player instance.
-	const player = new PodcastPlayer( id );
+	// Find DOM node.
+	const block = document.getElementById( id );
+	debug( 'constructing', id, block );
+
+	// Check if we can find the block.
+	if ( ! block ) {
+		return;
+	}
+
+	// Load data from the embedded JSON.
+	const dataContainer = block.querySelector( 'script[type="application/json"]' );
+	if ( ! dataContainer ) {
+		return;
+	}
+	let data;
+	try {
+		data = JSON.parse( dataContainer.text );
+	} catch ( e ) {
+		debug( 'error parsing json', e );
+		return;
```

Comment:
I wonder whether we ought to be surfacing a user-friendly error here. Even if it's just "there was a problem with this Block".

Reply:
I don't know here. Personally, it feels better to me to show the basic HTML list of links (our backend rendered one) rather than show an error which users have absolutely no power to fix.

Reply:
Ah yes. So the fallback would render anyway. In which case consider this resolved.